waifus/yuuki asuna.json

```diff
@@ -0,0 +1,11 @@
+{
+	"type" : 0,
+	"english-name" : "Yuuki Asuna",
+	"image" : "https://vignette.wikia.nocookie.net/swordartonline/images/8/88/Asuna_SAO.png/revision/latest?cb=20140301055612",
+	"japanese-name" : "結城 明日奈",
+	"year" : 2007,
+	"month" : 9,
+	"day-of-month" : 30,
+	"age-in-show": 18,
```

Comment:
I noticed that she ages though out the show, can you use the finally-legal-in-show variable to say when she becomes of legal age? https://github.com/yourWaifu/is-your-waifu-legal/blob/master/reference.md#story-variables